test-types/basic-matchers.test.ts

```diff
@@ -1,437 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- *
- * @type ./empty.d.ts
- */
-
-import {expectError, expectType} from 'mlh-tsd';
```

Comment:
will this file come back in a follow-up?

Reply:
Yes, we will be making new PR that adds test files.

Reply:
perfect!